includes/polling/os/packetfluxpfr.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * LibreNMS OS Polling module for packetflux
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+// Grab the coordinates read from GPS stats and update the location if different
+
+// PACKETFLUX-GNSS-MIB::gnssLatitude.0.0
+$latitude = snmp_get($device, '.1.3.6.1.4.1.32050.3.4.1.1.3.0.0','-OQv', 'PACKETFLUX-GNSS-MIB');
```

Comment:
These snmp_get should be converted to a multi call.

Reply:
Gave these a whirl.

Reply:
I reworked the logic here, two of the calls could be combined, The third uses a different MIB so from what I can gather it needs to be separate. My PHP is rusty, would I be on the right track here?